armi/reactor/reactors.py

```diff
@@ -49,8 +49,20 @@
 from armi.reactor.assemblyLists import SpentFuelPool
 from armi.reactor.flags import Flags
 from armi.reactor.systemLayoutInput import SystemLayoutInput
-from armi.settings.fwSettings.globalSettings import CONF_MATERIAL_NAMESPACE_ORDER
-from armi.settings.fwSettings.globalSettings import CONF_SORT_REACTOR
+from armi.settings.fwSettings.globalSettings import (
+    CONF_MATERIAL_NAMESPACE_ORDER,
+    CONF_SORT_REACTOR,
+    CONF_FRESH_FEED_TYPE,
+    CONF_TRACK_ASSEMS,
+    CONF_CIRCULAR_RING_PITCH,
+    CONF_AUTOMATIC_VARIABLE_MESH,
+    CONF_MIN_MESH_SIZE_RATIO,
+    CONF_DETAILED_AXIAL_EXPANSION,
+    CONF_GEOM_FILE,
+    CONF_NON_UNIFORM_ASSEM_FLAGS,
+    CONF_STATIONARY_BLOCK_FLAGS,
+    CONF_ZONE_DEFINITIONS,
+)
```

Comment:
This is admittedly a little bulky. Not terrible, but bulky. We could consider changing the API so that these setting defs live in their own file. That way we could just do something like:
```python
import armi.settings.fwSettings.globalSetting.definitions as defs
...
cs[defs.CONF_GEOM_FILE]
```
This has its own pros and cons. I'm definitely not attached to the idea. Just spitballing.